web-client/src/ustc-ui/DateInput/DateRangePickerComponent.tsx

```diff
@@ -39,16 +51,23 @@ export const DateRangePickerComponent = ({
       datePicker.on(startDatePickerRef.current);
       datePicker.on(endDatePickerRef.current);
       dateRangePicker.on(dateRangePickerRef.current);
+    } else if (
```

Comment:
What is the reason behind adding this else-if?

Reply:
....not sure. we don't have any recollection of adding it, though the git history clearly says we added it about three weeks ago. 

through brief testing with and without this block present, I have not observed much difference in functionality. I will remove it.